kong/custom/entity_crud.go

```diff
@@ -53,7 +53,7 @@ func render(template string, entity Entity) (string, error) {
 		if v := entity.GetRelation(m[1]); v != "" {
 			result = strings.Replace(result, m[0], v, 1)
 		} else {
-			return "", errors.New("cannot substitute '" + m[1] +
+			return "", fmt.Errorf("cannot substitute '" + m[1] +
```

Comment:
Would it be an opportunity to use verbs here?

Reply:
@mmorel-35 Mind sending a follow up PR to keep the scope small here?